### DISCUSSION TITLE
Unknown push status for Payments to QB Desktop

### DISCUSSION THREAD

**Discussion by hhelmric** [codatio/codat-docs#446] [Q&A]
Hi there -- I've recently started having a problem where payments pushed to Quickbooks Desktop are showing up with an `Unknown` status. I am able to push the invoice fine, but the payment always gives me that status. I was able to successfully push a payment before and am not sure what has changed. I've double checked all the data and references and it seems like Quickbooks should accept it. Is there any way I can further troubleshoot pushes that get this status?

**Comment by jmyorston** [comment]
Hi @hhelmric - It looks like you may have run into some timeouts issues, I have asked our support team to assist you with this.

**Reply by hhelmric** [reply]
Thanks -- I believe I can work with the `timeoutInMinutes` property? I'll give it a shot. I think I had seen there was a `TimedOut` status code, and since this was `Unknown` I assumed the issue was specifically not related to a timeout

**Comment by hhelmric** [comment]
<img width="584" alt="image" src="https://github.com/codatio/codat-docs/assets/25378854/498cafe8-2f91-4dfd-8b8f-e87859c03fbc">



 Looking at the Quickbooks Connector seems to have an issue. 

I restarted it and now it's showed as Idle, but it still won't accept any pushes.... 
I've also initiated a sync from the connector and it succeeded, but all pushes go to "Unknown"
